.github/workflows/test.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and Test
+
+on: push
+
+jobs:
+  manager:
+    name: Build and Test
+    
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # TODO run on Windows as well
+        os: [ubuntu-latest, macos-latest]
+      # Don't cancel jobs on other platforms if one fails
+      fail-fast: false
+      
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v1
+
+      - name: Install Node.js, NPM and Yarn
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Show Environment Info
```

Comment:
We're pinning to a particular Yarn version in the Travis builds. I'm glad that we could remove that!

👍